stories/editors.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { action } from "@storybook/addon-actions";
+import { EditorState, ContentState } from "draft-js";
+import { Editor, modifiers } from "../packages/aurora-editor";
+
+function stateFromText(text) {
+  return EditorState.createWithContent(ContentState.createFromText(text));
+}
+
+const canSubmit = () => {
+  const { canSubmit, canType } = modifiers;
+  const MyEditor = canType(canSubmit(Editor));
+  const message =
+    "Click on the text and press Shift-Enter to run a submit event.";
```

Comment:
I think this message might be in multiple places. Maybe modularize these UI message strings?

Reply:
Oh wait this is actually fine. This is just inside of storybook. It's just an example of how this UI component is used, it's not actually part of the project really. It's more like documentation.

Reply:
Run `npm run storybook` to see it. https://storybook.js.org/